src/media/css/rtl/footer.styl

```diff
@@ -0,0 +1,10 @@
+@import '../lib';
+
+@media $base-tablet {
+    +rtl() {
+        .footer p {
+            padding: 0 1em 0 0;
+            margin: 0 0 0 100px;
```

Comment:
I agree with you here but was trying to preserve the existing spacing...which seems a bit arbitrary anyway. With a flex-grow solution we wouldn't even need anything special for an RTL footer...it would flip automatically on dir=rtl